`test_fread_omnibus` fails due to void columns

For seed `2094287248` the error is
```
>               assert d0.ltypes == tuple(coltypes)
E               AssertionError: assert (ltype.void, ...type.str, ...) == (ltype.void, ...type.str, ...)
E                 At index 78 diff: ltype.void != ltype.str
E                 Full diff:
E                   (
E                    ltype.void,
E                    ltype.void,
E                    ltype.str,
E                    ltype.str,...
E                 
E                 ...Full output truncated (89 lines hidden), use '-vv' to show

tests/fread/test-fread-random.py:137: AssertionError
```

--- a/tests/fread/test-fread-random.py
+++ b/tests/fread/test-fread-random.py
@@ -49,10 +49,13 @@ def make_seeds():
     # If you want to test a specific seed, uncomment the following line:
     # return [398276719]
     n = 25
+    seeds = []
+    seeds += [2094287248] # issue 3278
     if (os.environ.get(root_env_name, "") != "" and
             os.environ.get(env_coverage, "") == ""):
         n = 100
-    return [random.randint(0, 2**31) for _ in range(n)]
+    seeds += [random.randint(0, 2**31) for _ in range(n)]
+    return seeds
 
 
 @pytest.mark.parametrize("seed", make_seeds())
@@ -158,7 +161,7 @@ def test_fread_omnibus(seed):
 #-------------------------------------------------------------------------------
 
 def all_voidlike(coldata):
-    return all(x in ['', '""', "''"] for x in coldata)
+    return all(x in ['', '""', "''", 'NA'] for x in coldata)
 
 
 def all_boollike(coldata):

